wifiphisher/pywifiphisher.py

```diff
@@ -746,6 +746,37 @@ def get_hostapd():
             '[' + R + '!' + W + '] Closing'
          ))
 
+
+def kill_interfering_procs(interfaces):
+    # Kill any possible programs that may interfere with the wireless card
+    # For systems with airmon-ng installed
+    if os.path.isfile('/usr/sbin/airmon-ng'):
+        proc = Popen(['airmon-ng', 'check', 'kill'], stdout=PIPE, stderr=DN)
+    # For ubuntu distros with nmcli
+    elif os.path.isfile('/usr/bin/nmcli') and \
+            os.path.isfile('/usr/sbin/rfkill'):
+        Popen(
+            ['nmcli', 'radio', 'wifi', 'off'],
+            stdout=PIPE,
+            stderr=DN
+        ).wait()
+        Popen(
+            ['rfkill', 'unblock', 'wlan'],
+            stdout=PIPE,
+            stderr=DN
+        ).wait()
+
+        time.sleep(1)
+
+        for interface in interfaces:
+            Popen(
+                ['ifconfig', interface, '10.15.0.1/24', 'up'],
```

Comment:
@lvrach Good.

@wraith-wireless Thanks for the notice. As soon as we have PyRIC imported, we'll switch to that.